### PR TITLE
feat(new-client): status service test(745)

### PR DESCRIPTION
**Pull request by slcfv3**

### CODE REVIEW

File: src/new-client/src/services/status.test.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    it("should show the service health again once the service is reconnected", async () => {
+      const mockStream = new BehaviorSubject(mockSource.a)
+      whenWatch("status", mockStream)
+      const { result } = renderUseStatus()
+      expect(result.current).toEqual(
+        alignResultTimestamp(mockResult.a, result.current),
+      )
+
+      await reactAct(async () => {
+        await wait(2001)
```
Comment (josepot) [inline src/new-client/src/services/status.test.tsx:132]: unfortunately this makes tests extremely slow to run, so could you try using [Jest Timer Mocks](https://jestjs.io/docs/en/timer-mocks) instead?

**Review by josepot — CHANGES_REQUESTED** [review]

File: src/new-client/src/services/status.test.tsx (hunk 1 above)
Comment (slcfv3) [reply src/new-client/src/services/status.test.tsx:132]: sure, I will try

**Review by josepot — CHANGES_REQUESTED** [review]
This is good stuff! But could you please create a `status` folder and make sure that we don't export the raw types publicly, basically follow the same approach that we used for the other services that have tests. Thanks!

**Comment by slcfv3** [comment]
> This is good stuff! But could you please create a `status` folder and make sure that we don't export the raw types publicly, basically follow the same approach that we used for the other services that have tests. Thanks!

no problem

**Review by josepot — APPROVED** [review]

**Pull request merged**